ansible/server-state-playbooks/nightshade-web/playbook.yml

```diff
@@ -0,0 +1,184 @@
+# Install OMERO.web with a public user on localhost
+
+- hosts: all
+
+  handlers:
+
+  # Handler for nginx
```

Comment:
The [`omero-common` role](https://github.com/openmicroscopy/ansible-role-omero-common/blob/master/handlers/main.yml) already includes a set of useful handlers, you should be able to use that instead of redefining one here.